### PR TITLE
Websocket multiple connection implementation with Goroutines

### PR DESCRIPTION
**Pull request by elraphty**
## Describe your changes

This feature enables the server to process multiple socket connections

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device

### CODE REVIEW

File: routes/index.go [hunk 1]
```diff
@@ -70,10 +71,8 @@ func NewRouter() *http.Server {
 	r.Group(func(r chi.Router) {
 		r.Get("/lnauth_login", handlers.ReceiveLnAuthData)
 		r.Get("/lnauth", handlers.GetLnurlAuth)
-		r.Get("/lnauth_poll", handlers.PollLnurlAuth)
 		r.Get("/refresh_jwt", handlers.RefreshToken)
 		r.Post("/invoices", handlers.GenerateInvoice)
-		r.Get("/invoices/{payment_request}", handlers.GetInvoiceStatus)
```
Comment (kevkevinpal) [inline routes/index.go:76]: frontend is still requesting this endpoint, so its not working

File: frontend/app/src/people/utils/connectCard.tsx [hunk 2]
```diff
@@ -73,15 +74,15 @@ export default function ConnectCard(props: ConnectCardProps) {
...
           <div className="bottomText">Scan or paste in Sphinx</div>
         </ModalBottomText>
+        {openAssignModal ? (<AssignBounty
+          dismiss={() => closeAssignModal()}
+          modalStyle={{ top: -64, height: 'calc(100% + 64px)' }}
+          person={person}
+          visible={openAssignModal}
+          created={created}
+          dismissConnectModal={props.dismiss}
+        />) : <></>}
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/connectCard.tsx:84]: you can use && instead of using the empty div

**Review by kevkevinpal — COMMENTED** [review]
looks good to me can you also run

`go get github.com/gobuffalo/packr/v2/packr2/cmd@v2.8.1` cause we need that to build the frontend

**Comment by elraphty** [comment]
> @elraphty is the `HOST` env var new?

I can't seem to find the HOST env, do you mean r.Host or config.Host?

**Comment by kevkevinpal** [comment]
> > @elraphty is the `HOST` env var new?
> 
> I can't seem to find the HOST env, do you mean r.Host or config.Host?

I was meaning the config.Host but that might have been added before this PR but I just never seen that env var before

File: routes/index.go (hunk 1 above)
Comment (kevkevinpal) [reply routes/index.go:76]: looks like this was an issue with me deploying without packr2 build, marking resolved

File: frontend/app/src/people/utils/connectCard.tsx [hunk 3]
```diff
@@ -74,14 +75,16 @@ export default function ConnectCard(props: ConnectCardProps) {
...
         </ModalBottomText>
-        {openAssignModal ? (<AssignBounty
-          dismiss={() => closeAssignModal()}
-          modalStyle={{ top: -64, height: 'calc(100% + 64px)' }}
-          person={person}
-          visible={openAssignModal}
-          created={created}
-          dismissConnectModal={props.dismiss}
-        />) : <></>}
+        <>
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/connectCard.tsx:78]: do you still need these? <></>

**Pull request merged**